tests/network/general/test_ip_family_services.py

```
@@ -0,0 +1,203 @@
+    )
+
+
+@pytest.fixture()
+def default_ip_family_policy_service(running_vm_for_exposure):
+    running_vm_for_exposure.custom_service_enable(
+        service_name="default-ip-family-policy-svc",
+        port=SSH_PORT_22,
+    )
+
+
+def _create_virtctl_expose_service(
```

Ruff (0.14.13)

Missing return type annotation for private function `_create_virtctl_expose_service`

(ANN202)